lib/cisco_node_utils/bfd_global.rb

```diff
@@ -0,0 +1,121 @@
+#
+# NXAPI implementation of BFD Global class
+#
+# May 2016, Sai Chintalapudi
+#
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # node_utils class for bfd_global
+  class BfdGlobal < NodeUtil
+    attr_reader :name
+
+    def initialize(name, instantiate=true)
+      fail ArgumentError unless name.to_s == 'default'
```

Comment:
Why do we need a `name` parameter?  Personally, I would treat this like vtp.

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/vtp.rb#L28